lib/superstatic.js

```diff
@@ -19,6 +19,17 @@ var loadConfigFile = require('./loaders/config-file');
 
 var CWD = process.cwd();
 
+  // shink-ray is a fork of compression which supports advanced compression encodings but not ES5.
+function initializeCompressor(options) {
+  var compressor;
```

Comment:
...and on further thought I think you should probably actually just do this at the top of the file, then you don't need `initializeCompressor` you can just use the var.